app/controllers/tweets_controller.rb

```diff
@@ -0,0 +1,71 @@
+class TweetsController < ApplicationController
+
+  get '/tweets' do
+    if !!session[:id]
+      @user = User.find(session[:id])
+      erb :'tweets/tweets'
+    else
+      redirect to "/login"
+    end
+  end
+
+  get '/tweets/new' do
+    if !session[:id]
+      redirect to "/login"
+    else
+      erb :'tweets/create_tweet'
+    end
+  end
+
+  post '/tweets' do
+    if params.has_value?("")
+      redirect to "/tweets/new"
+    else
+      @user = User.find(session[:id])
+      @tweet = Tweet.create(params)
+      @user.tweets << @tweet
```

Comment:
Another way to do this is to make use of the `has_many` relationship on User:
```
@tweet = current_user.tweets.create(content: params[:content])
```
Both approaches are good, but in some cases you may find this option useful.